### PR TITLE
chore: update ruff to 0.6.4 and update docs to fix error D417

### PR DESCRIPTION
**Pull request by dmitryduev**
Description
-----------
- Several docstrings had missing argument descriptions.
- There was a bug in how last lines of docstrings were treated in generate_stubs.py.

- [x] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

File: wandb/__init__.pyi [hunk 1]
```diff
@@ -405,7 +412,10 @@ def finish(exit_code: Optional[int] = None, quiet: Optional[bool] = None) -> Non
 
     Arguments:
         exit_code: Set to something other than 0 to mark a run as failed
-    quiet: Set to true to minimize log output
+        quiet: Set to true to minimize log output
+
+    Returns:
+    None
```
Comment (kptkin) [inline wandb/__init__.pyi:418]: don't need

File: wandb/__init__.pyi [hunk 2]
```diff
@@ -816,7 +827,10 @@ def agent(
         project: The name of the project where W&B runs created from
             the sweep are sent to. If the project is not specified, the
             run is sent to a project labeled "Uncategorized".
-    count: The number of sweep config trials to try.
+        count: The number of sweep config trials to try.
+
+    Returns:
+    None
```
Comment (kptkin) [inline wandb/__init__.pyi:833]: same

**Review by kptkin — APPROVED** [review]
thanks for the fix, it's not required when the function returns None

File: wandb/__init__.pyi (hunk 1 above)
Comment (dmitryduev) [reply wandb/__init__.pyi]: it was a hack to "fix" the padding on the last line of the docstring, but I fixed the underlying issue instead.

Comment (dmitryduev) [reply wandb/__init__.pyi]: I kno, I removed it

**Pull request merged**